build.sc

```diff
@@ -92,7 +91,7 @@ trait MillPublishModule extends PublishModule{
   def javacOptions = Seq("-source", "1.8", "-target", "1.8")
 }
 trait MillApiModule extends MillPublishModule with ScalaModule{
-  def scalaVersion = T{ "2.12.10" }
+  def scalaVersion = T{ "2.13.1" }
```

Comment:
Can we move this up to `MillPublishModule` or into a val in `Deps` object?

Reply:
Sure, we could, but for this PR I'm trying to preserve the existing file structure as much as possible

Reply:
Fine.